### PR TITLE
Added 'default' color

### PR DESCRIPTION
**Pull request by jaytaph**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #15246 |
| License | MIT |
| Doc PR |  |

Adding a "default" ansi color which corresponds to 39 and 49.

### CODE REVIEW

File: src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php [hunk 1]
```diff
@@ -29,6 +29,7 @@ class OutputFormatterStyle implements OutputFormatterStyleInterface
         'magenta' => array('set' => 35, 'unset' => 39),
         'cyan' => array('set' => 36, 'unset' => 39),
         'white' => array('set' => 37, 'unset' => 39),
+        'default' => array('set' => 39, 'unset' => 39),
```
Comment (fabpot) [inline src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php]: Wouldn't it be better to name it `transparent`?

**Comment by fabpot** [comment]
That's probably a bug that should be part of 2.3.

File: src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php (hunk 1 above)
Comment (xabbuh) [inline src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php]: I think `default` is the right term (it depends on the user's settings whether this is transparent, black, white, or something else).

**Comment by xabbuh** [comment]
:+1: (and I'm also in favor of treating this as a bug)

File: src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php (hunk 1 above)
Comment (jaytaph) [inline src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php]: According to the standard, it's indeed "default": http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-048.pdf (8.3.117)

**Comment by jaytaph** [comment]
Not sure if i need to rebase this against other branches, but ping me when something is needed that i can do.

**Comment by fabpot** [comment]
Thank you @jaytaph.

**Pull request closed without merging**